content/en/security/application_security/enabling/single_step/threat_detection.md

```diff
@@ -209,6 +272,25 @@ Run the following commands and restart the service to stop injecting the library
 3. To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
 {{% /tab %}}
 
+{{% tab "Kubernetes" %}}
+
+1. Set the `admission.datadoghq.com/enabled:` label to `"false"` for the pod spec:
+   ```yaml
+   spec:
+     template:
+       metadata:
+         labels:
+           admission.datadoghq.com/enabled: "false"
+   ```
+2. Apply the configuration:
+   ```bash
+   kubectl apply -f /path/to/your/deployment.yaml
+   ```
+3. Restart the services you want to remove instrumentation for.
+
+**Note:** You can disable ASM while keeping APM up by adding the `DD_APPSEC_ENABLED=false` environment variable to your deployments.
```

Comment:
```suggestion
1. Set the `admission.datadoghq.com/enabled:` label to `"false"` for the pod spec:
     
   ```yaml
   spec:
     template:
       metadata:
         labels:
           admission.datadoghq.com/enabled: "false"
   ```
2. Apply the configuration:
   
   ```bash
   kubectl apply -f /path/to/your/deployment.yaml
   ```
3. Restart the services you want to remove instrumentation for.

<div class="alert alert-info"> You can disable ASM while keeping APM up by adding the <code>DD_APPSEC_ENABLED=false</code> environment variable to your deployments.</div>
```